### ISSUE TITLE
This won't run in a properly UAC'ed windows setting

### ISSUE THREAD

**Issue by Pomax** [nephics/noxmox#8]
windows with proper user control settings requires admin privileges to write anything to root, including the `c:\tmp\nox` dir that noxmox tries to make. Instead, on the win32 platform there is the `TEMP` var that indicates the current user's temp folder location, so you could do something like

```
if (!options.prefix) {
  if (process.platform === "win32") {
    options.prefix = process.env["TEMP"] + "/mox";
  } else {
    options.prefix = '/tmp/mox';
  }
}
```

on https://github.com/nephics/noxmox/blob/master/mox.js#L84 (windows doesn't care about `\` vs `/` and will happily navigate and serve data from locations that mix the two, or even exclusively use `/`)

**Comment by Pomax** [comment]
https://github.com/nephics/noxmox/pull/10 should address this issue

**Comment by nephics** [comment]
Fixed by pull request #10.
